### PR TITLE
Supplemental Security Income for individuals

### PR DESCRIPTION
**Pull request by MaxGhenis**

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Despite the variety of broken things, I think this is actually fairly close, aside from deeming rules. @nmrodelo

**Comment by MaxGhenis** [comment]
I'm lost by the test errors. They seem to be breaking because SSI is person-level instead of SPM unit-level as it was previously, but it's breaking in `snap_unearned_income`, which calls `aggr(spm_unit, period, sources)` over a yaml file of all person-level income sources:
```yaml
    - dividend_income
    - interest_income
    - gi_cash_assistance
    - social_security
    - ssi
```

File: openfisca_us/variables/ssa/ssi/ssi.py [hunk 1]
```diff
@@ -3,15 +3,21 @@
...
-        return where(eligible, amount_if_eligible, 0)
+    def formula(person, period, parameters):
+        abd = person("is_ssi_aged_blind_disabled", period)
+        countable_resources = person("ssi_countable_resources", period)
+        p_ssi = parameters(period).ssa.ssi
+        # Only individual is modeled currently.
+        resource_limit = p_ssi.eligibility.resources.limit.individual
+        meets_resource_test = countable_resources <= resource_limit
+        # Calculate amount.
+        amount = parameters(period).ssi.amount.individual
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/ssa/ssi/ssi.py:20]: The error message obfuscated this, but here's the issue:
```suggestion
        amount = parameters(period).ssa.ssi.amount.individual
```

**Comment by nikhilwoodruff** [comment]
Fixed all the immediate bugs, but left with this policy test:
```console
ssi<2022> >> [0.]
    is_ssi_aged_blind_disabled<2022> >> [False]
      is_ssi_aged<2022> >> [False]
        age<2022> >> [18.]
      is_blind<2022> >> [False]
      is_ssi_disabled<2022> >> [False]
    ssi_countable_resources<2022> >> [0.]
    ssi_countable_income<2022> >> [0.]
      ssi_earned_income<2022> >> [0.]
        employment_income<2022> >> [0.]
        self_employment_income<2022> >> [0.]
      ssi_unearned_income<2022> >> [10.]
        dividend_income<2022> >> [0.]
        interest_income<2022> >> [0.]
        gi_cash_assistance<2022> >> [0.]
        social_security<2022> >> [10.]
          social_security_dependents<2022> >> [1.]
          social_security_disability<2022> >> [2.]
          social_security_retirement<2022> >> [3.]
          social_security_survivors<2022> >> [4.]
      ssi_deemed_income<2022> >> [0.]
F

===================================================================================== FAILURES =====================================================================================
___________________________________________________________________________________ test session ___________________________________________________________________________________
/home/nikhil/policyengine/openfisca-us/openfisca_us/tests/policy/baseline/ssa/social_security.yaml:
  Test 'Social Security sums dependent, disability, retirement, and survivors benefits':
    ssi@2022: [0.] differs from 10.0 with an absolute margin [10.] > 0
```

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**